client/components/Orders/Orders.jsx

```diff
@@ -1,44 +1,17 @@
 import React from 'react';
 import './Orders.scss';
 import PropTypes from 'prop-types';
-import OrderStage from '../OrderStage/OrderStage';
+/* import OrderStage from '../OrderStage/OrderStage'; */
+import OrderStage from '../../containers/OrderStage';
 
 function Orders(props) {
-  const { stages, onClose, discounts } = props;
+  const { stages, price, onClose, discounts } = props;
 
   const stagesList = stages.map((elem) => {
-    const { orders, title, _id } = elem;
-
-    const time = Math.max(...orders.map((order) => order.time));
-    const diff = Math.floor((Date.now() - time) / 1000);
-
-    const intervals = [
-      [3600, ' hours'],
-      [60, ' minutes'],
-      [10, '0 seconds'],
-    ];
-    const stringParams = intervals.reduce(
-      (accumulator, currentValue) =>
-        currentValue[0] <= accumulator[0] && accumulator[1] === 'just now'
-          ? currentValue
-          : accumulator,
-      [diff, 'just now']
-    );
-
-    const timeString =
-      stringParams[1] === 'just now'
-        ? stringParams[1]
-        : `${Math.floor(diff / stringParams[0])}${stringParams[1]} ago`;
-
-    return <OrderStage time={timeString} orders={orders} title={title} id={_id} />;
+    const Stage = OrderStage(elem._id);
```

Comment:
@IgorChaikin It it possible to avoid calling function `OrderStage`, which returns components, based on `elem._id`), and use just component with props instead

```js
const stagesList = stages.map((elem) => <Stage {...someProps} />)`
```